forc-plugins/forc-client/src/ops/deploy/cmd.rs

```diff
@@ -69,4 +69,13 @@ pub struct DeployCommand {
     /// Output the time elapsed over each part of the compilation process.
     #[clap(long)]
     pub time_phases: bool,
+    /// Do not sign the transaction
+    #[clap(long)]
+    pub not_sign: bool,
```

Comment:
Was just taking a quick look at #2656 and noticed this flag - perhaps a better name might be `unsigned`?